docs/documentation/_data/deckhouse-alerts.yml

```diff
@@ -15,9 +15,9 @@ alerts:
       moduleUrl: 101-cert-manager
       module: cert-manager
       edition: ce
-      description: Certificate {{$labels.exported_namespace}}/{{$labels.name}} expired
+      description: Certificate {{$labels.exported_namespace}}/{{$labels.name}} expired or has False condition due to another reason.
       summary: |
-        Certificate expired
```

Comment:
What does it mean? Are there other reasons the condition can be False? Telling user that the condition is False is to broad, we have to be specific in alerts.

Reply:
The reasons are partly described in #9582.

The main problem with the current alerts is that they light up not only when the certificate has expired. It also lights up when the certificate could not be ordered at all (and went into the `False` state). cert-manager does not have an additional label in the metric to distinguish these states.
As a quick fix, it was suggested to at least change the text of the alert to avoid misleading users about the reason for its triggering. For this reason, the alert description does not indicate the exact reason

Reply:
according to the code, the manager either sets the expiry metric to 0.0 or to the actual date from `NotAfter` status field
```
// updateCertificateExpiry updates the expiry time of a certificate
func (m *Metrics) updateCertificateExpiry(crt *cmapi.Certificate) {
        expiryTime := 0.0

        if crt.Status.NotAfter != nil {
                expiryTime = float64(crt.Status.NotAfter.Unix())
        }
```
could we alter the alert definition to exclude 0.0 value from matching the `< 1209600` condition  and introduce a new alert like `certificate isn't provided` or whatsoever for the `0,0` case?

Reply:
no, a description like 'Certificate has False condition' is too obscure.

Reply:
Okay, I'll change the alert title to the suggested option `certificate isn't provided`

Reply:
Done

Reply:
uh, it seems you didn't get my idea :(
I suggested having not one, but two alerts:
the first one, for the certificates that are about to expire - we check that reported value `< 1209600` and `!= 0.0` in this alert, it's description should stay as it is now.
the second one, that checks that the metric reports `0.0` value and it should have a description explaining that the certificate `wasn't provided` or whatsoever.
This way we have two not crossing conditions and two distinctive alerts.
Does it make sense?

Reply:
Run `make generate` and rebase changes